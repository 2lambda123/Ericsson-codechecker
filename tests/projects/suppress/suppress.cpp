int main()
{
    int *p = 0;

    // codechecker_suppress [all] multi line
    // comments!

    sizeof(44);

    // codechecker_suppress [all] deliberate warning!

    sizeof(43);


    // No suppress comment.
    sizeof(42);

    // codechecker_suppress [all] deliberate segfault!
    return *p + 2;
}
