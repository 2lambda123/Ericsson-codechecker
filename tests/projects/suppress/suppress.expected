c417110e71a5cb8e8163d524076bb3ed||suppress.cpp||deliberate segfault!
5f45decbfcfd4374ea7febf6227c96a7||suppress.cpp||deliberate warning!
090ab5162ed1167c16b9a099cc7cc382||suppress.cpp||multi line comments!
